R/Matched.R

```
@@ -490,6 +490,25 @@ setMethod("matchedData", "Matched", function(object,
   }
 }
 
+.dollar2 <- function(query, target, matches, name) {
+  if(name %in% .colnames(query, target, matches))
```

maybe it would make sense to have then also an `else stop("'", name, "' not available")` or something like this.

---

Ok, I will add it. I didn't because if I remember correctly when $ is called (for example on a data.frame) with a name that doesn't exist it just returns NULL

---

OK, I see. But for the `Matchedxxx` objects I would prefer to throw an error in such a case.